### PR TITLE
Add episode 43

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _episodes/43.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+## Hosts
+
+* Brian Olsen, Developer Advocate at [Starburst](https://starburst.io)
+  ([@bitsondatadev](https://twitter.com/bitsondatadev))
+* Manfred Moser, Director of Information Engineering at 
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
```
Comment (mosabua) [inline _episodes/43.md:43]: empty line after

File: _episodes/43.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+solutions can make a tremendous impact on the latency and cost of your queries.
+
+### Trino without caching
+
+There seems to be a sizeable portion of the community who aren't using a
+caching solution. Not all workloads will really benefit from caching. If you
+are performing more writes than reads, the cache will need to constantly be
+invalidated before performing each read. If you are scanning all your data to
+run daily migrations, you would not benefit from caching. However, one of the
+most common use cases where Trino shines is interactive adhoc analysis. This
```
Comment (mosabua) [inline _episodes/43.md:65]: analytics

File: _episodes/43.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+## Concept of the episode: Data caching and orchestration
+
+Out of all those petabytes of data you store, only a small fraction of it is
+creating business value for you today. When you scan the same data multiple
+times and transfer it over the wire, you're wasting time, compute cycles, and
+ultimately money. This gets worse when you're pulling data across regions or
+clouds from disaggregate Trino clusters. In situations like these, caching
+solutions can make a tremendous impact on the latency and cost of your queries.
+
+### Trino without caching
```
Comment (mosabua) [inline _episodes/43.md:58]: we need to add that this is about caching for object files.. and therefore only applies to object storage connectors (hive, iceberg, delta lake and hudi)

File: _episodes/43.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+formats like Iceberg. 
+
+### Hive connector caching
+
+Trino has an [embedded caching engine](https://trino.io/docs/current/connector/hive-caching.html)
+in the Hive connector. This is convenient as it ships with Trino, however, it 
+does not work outside the Hive connector. The caching engine is 
+[Rubix](https://github.com/qubole/rubix) which has not been developed over two
+years. While this system works for simple Hive use cases, it fails to address
+use cases outside of Hive and hasn't been maintained since 2020. There many
```
Comment (mosabua) [inline _episodes/43.md:76]: There are

File: 404.md [hunk 5]
```diff
@@ -1,10 +0,0 @@
----
```
Comment (mosabua) [inline 404.md:1]: Why are you removing this page? I think we still need it.

File: _episodes/43.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+In this episode, we will focus on the latter type of caching. This will apply to
+connectors like Hive, Iceberg, Delta Lake, and Hudi.
+
+### Hive connector caching
+
+Trino has an [embedded caching engine](https://trino.io/docs/current/connector/hive-caching.html)
+in the Hive connector. This is convenient as it ships with Trino, however, it 
+does not work outside the Hive connector. The caching engine is 
+[Rubix](https://github.com/qubole/rubix) which has not been developed over two
```
Comment (mosabua) [inline _episodes/43.md:105]: Rubix. 

remove the rest since it comes up in the ext sentence.

File: _episodes/43.md [hunk 7]
```diff
@@ -0,0 +1,200 @@
...
+years. While this system works for simple Hive use cases, it fails to address
+use cases outside of Hive and hasn't been maintained since 2020. There are many
+features missing like kubernetes and containerization, security features,
```
Comment (mosabua) [inline _episodes/43.md:108]: remove "Kubernetes and containerization" and rework the rest as needed.

File: _episodes/43.md [hunk 8]
```diff
@@ -0,0 +1,200 @@
...
+support for more compute engines. 
+
+### What is Alluxio?
+
+Alluxio is world’s first open source data orchestration technology for analytics
+and AI for the cloud. It provides a common interface enabling computation
+frameworks to connect to numerous storage systems through a common interface.
+Alluxio’s memory-first tiered architecture enables data access at speeds orders
+of magnitude faster than existing solutions. Alluxio was originally developed at
+Berkley Amp Labs, [formerly known as Tachyon](
```
Comment (mosabua) [inline _episodes/43.md:118]: Amp Labs, and was originally called Tachyon. It was less focussed ...

File: _episodes/43.md [hunk 9]
```diff
@@ -0,0 +1,200 @@
...
+https://amplab.cs.berkeley.edu/wp-content/uploads/2014/11/2014_socc_tachyon.pdf),
+and was less focused on caching and data orchestration and more focused on
+fault-tolerance via lineage and other techniques borrowed from Spark. 
+
+Alluxio lies between data driven applications, such as Trino and Apache Spark,
+and various persistent storage systems, such as Amazon S3, Google Cloud Storage,
+HDFS, Ceph, and Minio. Alluxio unifies the data stored in these different
+ storage systems, presenting unified client APIs and a global namespace to its
```
Comment (mosabua) [inline _episodes/43.md:126]: remove space in from of "storage systems.

File: _episodes/43.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
...
+upper layer data driven applications.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/43/alluxio-architecture.png"/>
+</p>
+
+Alluxio is commonly used as a distributed shared caching service so compute
+engines talking to Alluxio can transparently cache frequently accessed data,
+especially from remote locations, to provide in-memory I/O throughput. Alluxio
+also makes using Object storage much simpler as you will not hit the storage
```
Comment (mosabua) [inline _episodes/43.md:136]: object

File: _episodes/43.md [hunk 11]
```diff
@@ -0,0 +1,200 @@
...
+in the cloud if the file is already cached by Alluxio. Alluxio also handles the
+complex mapping client interface requests to underlying storage interfaces.
```
Comment (mosabua) [inline _episodes/43.md:138]: mapping of client


but also .. I dont understand what this means.

File: _episodes/43.md (hunk 10 above)
Comment (mosabua) [reply _episodes/43.md:136]: also .. this sentence feels wrong ... it doesnt really make things simpler does it?

File: _episodes/43.md [hunk 12]
```diff
@@ -0,0 +1,200 @@
...
+
+Source: <https://docs.alluxio.io/os/user/stable/en/Overview.html>
+
+### What is data orchestration?
+
+A data orchestration platform abstracts data access across storage systems,
+virtualizes all the data, and presents the data via standardized APIs with
+global namespace to data-driven applications. In the meantime, it should have
+caching functionality to enable fast access to warm data. In summary, a data
+orchestration platform provides data-driven applications Data Accessibility,
```
Comment (mosabua) [inline _episodes/43.md:152]: get rid of the title case for all these ..

File: _episodes/43.md [hunk 13]
```diff
@@ -0,0 +1,200 @@
...
+### Trino and Alluxio: Expedia use case
+
+Expedia needed to have the ability to query cross cluster over different regions
+while simplifying the interface to their local data sources.
+
+<p align="center">
+<img align="center" width="100%" height="100%" src="/assets/episode/43/expedia-trino-alluxio.png"/>
+</p>
+
+Source: [Unifying Cross-region Access in the Cloud at Expedia Group — The Path Toward Data Mesh in the Brand World](https://www.alluxio.io/blog/unifying-cross-region-access-in-the-cloud-at-expedia-group-the-path-toward-data-mesh-in-the-brand-world/)
```
Comment (mosabua) [inline _episodes/43.md:166]: I know its a source.. but get rid of the title case anyway .. its not readable here

File: _episodes/43.md [hunk 14]
```diff
@@ -0,0 +1,200 @@
...
+
+## PR of the episode: Alluxio/alluxio PR 13000 Add a doc for Trino
+
+[This episode's PR](https://github.com/Alluxio/alluxio/pull/13000) is actually
+one of those not located in the Trino repository. This PR comes from the Alluxio
```
Comment (mosabua) [inline _episodes/43.md:171]:
```suggestion
not located in a Trino repository. This PR comes from the Alluxio
```

File: _episodes/43.md [hunk 15]
```diff
@@ -0,0 +1,200 @@
...
+like this helped continue the Trino community as it grew awareness around the
+new name, as well as, fixed any potential issues that occurred with the hasty
+renaming we had to do.
+
+This was submitted by Alluxio engineer, [David Zhu](https://github.com/yuzhu).
+A huge thanks to David and his contributions to Trino as well!
+
+## Demo of the episode: Running Trino on Alluxio
+
+This demo of the episode, covers how to configure Alluxio to use write through
```
Comment (mosabua) [inline _episodes/43.md:182]: write-through

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _episodes/43.md (hunk 10 above)
Comment (bitsondatadev) [reply _episodes/43.md:136]: I'll avoid using "simpler" but rather specify that "they enable unifying all data sources to a single file namespace". This makes things simpler in some aspects if you have multiple object storages across different storage technologies, object storage in different regions, or stored across different clouds.

File: _episodes/43.md (hunk 11 above)
Comment (bitsondatadev) [reply _episodes/43.md:138]: Going to reword [with this comment in mind](https://github.com/trinodb/trino.io/pull/391#discussion_r1058481809).

File: 404.md (hunk 5 above)
Comment (bitsondatadev) [reply 404.md:1]: WTF lol. I don't know what happened

**Review by mosabua — APPROVED** [review]
Looks good now

File: _episodes/43.md [hunk 16]
```diff
@@ -0,0 +1,199 @@
...
+Alluxio’s memory-first tiered architecture enables data access at speeds orders
+of magnitude faster than existing solutions. Alluxio was originally developed at
+Berkley Amp Labs, [and was originally called Tachyon](
+https://amplab.cs.berkeley.edu/wp-content/uploads/2014/11/2014_socc_tachyon.pdf).
+It was less focused on caching and data orchestration and more focused on
+fault-tolerance via lineage and other techniques borrowed from Spark. 
+
+Alluxio lies between data driven applications, such as Trino and Apache Spark,
+and various persistent storage systems, such as Amazon S3, Google Cloud Storage,
+HDFS, Ceph, and Minio. Alluxio unifies the data stored in these different
```
Comment (mosabua) [inline _episodes/43.md:124]: isnt it MinIO ?

**Comment by bitsondatadev** [comment]
:shipit:

**Pull request merged**